demos/coreMQTT/CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
-# AFR coreMQTT demos
+# AFR coreMQTT demo
 afr_demo_module(core_mqtt)
 
 # Set the default demo that is enabled in the downloaded package.
-afr_set_demo_metadata(ID "CORE_MQTT_MUTUAL_AUTH_DEMO")
+afr_set_demo_metadata(ID "CORE_MQTT_BASIC_TLS_DEMO")
```

Comment:
This change shouldn't be done as we want the default enabled demo to be **mutual auth**